src/components/medication-dispense-review.component.tsx

```diff
@@ -404,7 +404,6 @@ const MedicationDispenseReview: React.FC<MedicationDispenseReviewProps> = ({
           disabled={!userCanModify}
           hideSteppers={true}
           id="dosingQuanity"
-          light={isTablet}
```

Comment:
I got a warning that this was no longer supported on this element (I think a NumberInput?) so I just removed it.